pkg/test/framework/components/echo/config.go

```diff
@@ -71,6 +71,9 @@ type Config struct {
 	// Headless (k8s only) indicates that no ClusterIP should be specified.
 	Headless bool
 
+	// StatefulSet indicates that
+	StatefulSet bool
```

Comment:
In `common.FillInDefaults`, if `StatefulSet` is true, we should automatically set `Headless` as well.